_overviews/scala3-book/methods-most.md

```diff
@@ -269,7 +278,7 @@ For more details on the `Matchable` trait, see the [Reference documentation][ref
 
 ## Controlling visibility in classes
 
-In classes, objects, traits, and enums, Scala methods are public by default, so the `Dog` instance created here can access the `speak` method:
+In classes, objects, traits, and enums, Scala methods are public by default, so world can access the `speak` method of `Dog` instance created here:
```

Comment:
yes, the previous version was wrong. But I think the whole section needs a rewrite, not tweaks about what a cat says.

Reply:
and?